src/components/AccessControlNew.tsx

```diff
@@ -220,12 +260,50 @@ const AccessControlNew = () => {
                                                     <Input placeholder="Add rule name..." ref={inputNameRef} onPressEnter={() => toggleEditName(false)} onBlur={() => toggleEditName(false)} autoComplete="off"/>
                                                 </Form.Item>
                                             )}
+                                            { !editDescription ? (
+                                                <div className={"access-control input-text ant-drawer-subtitle"} onClick={() => toggleEditDescription(true)}>{formRule.Description && formRule.Description.trim() !== "" ? formRule.Description : 'Add description...'}</div>
+                                            ) : (
+                                                <Form.Item
+                                                    name="Description"
+                                                    label={null}
+                                                    style={{marginTop: 24}}
+                                                >
+                                                    <Input placeholder="Add description..." ref={inputDescriptionRef} onPressEnter={() => toggleEditDescription(false)} onBlur={() => toggleEditDescription(false)} autoComplete="off"/>
```

Comment:
Codacy has a fix for the issue: [Replace `··························<Input·placeholder="Add·description..."·ref={inputDescriptionRef}·onPressEnter={()·=>·toggleEditDescription(false)}·onBlur={()·=>·toggleEditDescription(false)}·autoComplete="off"/>` with `<Input⏎····························placeholder="Add·description..."⏎····························ref={inputDescriptionRef}⏎····························onPressEnter={()·=>·toggleEditDescription(false)}⏎····························onBlur={()·=>·toggleEditDescription(false)}⏎·················](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
                          <Input
```